src/Sylius/Bundle/ApiBundle/StateProvider/Shop/Account/VerifyShopUser/ItemProvider.php

```diff
@@ -11,29 +11,31 @@
 
 declare(strict_types=1);
 
-namespace Sylius\Bundle\ApiBundle\StateProvider\Input;
+namespace Sylius\Bundle\ApiBundle\StateProvider\Shop\Account\VerifyShopUser;
```

Comment:
Similar as with the admin side:
```suggestion
namespace Sylius\Bundle\ApiBundle\StateProvider\Shop\VerifyShopUser;
```
or
```suggestion
namespace Sylius\Bundle\ApiBundle\StateProvider\Shop\ShopUser\VerifyShopUser;
```
but I'm not 100% convinced 🤔

Reply:
Same here